scss/_patterns_code-snippets.scss

```diff
@@ -0,0 +1,48 @@
+// Code and pre styles
+@mixin vf-p-code-snippets {
+
+  .code-snippet {
+    position: relative;
+    overflow: hidden;
+    transition: all .2s;
+    border: 1px solid $color-mid-grey;
+    border-radius: 4px;
+    background-color: $color-light;
+    font-family: $font-monospace;
+
+    &__input {
+      font-size: 1em;
+      font-weight: 300;
+      width: 100%;
+      padding: .7em 1em;
+      color: $color-cool-grey;
+      border: 0;
+      background: transparent;
+    }
+
+    &__action {
+      position: absolute;
+      top: 0;
+      right: 0;
+      display: block;
+      width: 50px;
+      height: 100%;
+      text-indent: -9999px;
+      border-left: 1px solid $color-mid-grey;
+      background-color: $color-light;
+      background-image: url('#{$assets-path}994e60f9-get-link-url_16.svg');
```

Comment:
This just uses the asset that was already there. It has not been a defined requirement up until this point that Vanilla assets should all be included from within the framework - there is at least ten other places where assets are pulled from the assets server.

Can you create a Proposal and we'll discuss this in the working group? If agreed, it seems like a good task top pick up in the pre-launch sprint.